designer/import_dialog.ui

```diff
@@ -40,6 +40,23 @@
       </widget>
      </item>
      <item row="4" column="0">
+      <widget class="QLabel" name="label">
+       <property name="text">
+        <string>&amp;Author</string>
```

Comment:
This access key collides with "add notes." Unfortunately all of the other letters conflict as well. I'm thinking the least evil would be to change the key for "add notes" to `d` -- any thoughts? Most likely people will have been using Enter to accept the dialog anyway.

Reply:
I think changing the key for "Add notes" to "d" is a good compromise.